src/renderers/native/__tests__/ReactNativeMount-test.js

```diff
@@ -45,17 +47,17 @@ describe('ReactNative', () => {
 
     ReactNative.render(<View foo="foo" />, 11);
 
-    expect(UIManager.createView.mock.calls.length).toBe(2);
-    expect(UIManager.setChildren.mock.calls.length).toBe(2);
+    expect(UIManager.createView.mock.calls.length).toBe(1);
```

Comment:
I see changes to this test but no changes to how stack works. So how did this pass before vs. now?